packages/wdio-sauce-service/README.md

```diff
@@ -148,16 +148,23 @@ Default: `false`
 *(only for vm and or em/simulators)*
 
 ### sauceConnectOpts
-Apply Sauce Connect options (e.g. to change port number or logFile settings). See [this list](https://wiki.saucelabs.com/display/DOCS/Sauce+Connect+Proxy+Command-Line+Quick+Reference+Guide) for more information. Per default the service disables SC proxy auto-detection as via `noAutodetect` as this can be unreliable for some machines.  
+Apply Sauce Connect options (e.g. to change port number or logFile settings). See [this list](https://wiki.saucelabs.com/display/DOCS/Sauce+Connect+Proxy+Command-Line+Quick+Reference+Guide) for more information. Per default the service disables SC proxy auto-detection as via `noAutodetect` as this can be unreliable for some machines.
 
-NOTE: When specifying the options the `--` should be omitted.  
+NOTE: When specifying the options the `--` should be omitted.
 It can also be turned into camelCase (e.g. `shared-tunnel` or `sharedTunnel`).
 
 Type: `Object`<br />
 Default: `{ noAutodetect: true }`
 
 *(only for vm and or em/simulators)*
 
+### uploadLogs
+
+If `true` this options uploads all WebdriverIO log files to the Sauce Labs platform for further inspection.
```

Comment:
Just one last remark, this will only work if the logs are written to a file, but I don't see it in the docs here. If you would add it here then it's good to go